todo_app/app.py

```diff
@@ -1,20 +1,52 @@
 from flask import Flask, render_template, session, request, redirect
 from todo_app.flask_config import Config
 from todo_app.data.session_items import _DEFAULT_ITEMS, get_items, add_item
+import requests
+import os
 
 app = Flask(__name__)
 app.config.from_object(Config)
+TRELLO_KEY = os.environ.get('TRELLO_KEY')
+TRELLO_TOKEN = os.environ.get('TRELLO_TOKEN')
+TRELLO_LIST_ID_DOING = "609392e0aaa6e8618e341f90"
+TRELLO_LIST_ID_TODO = "609392e0aaa6e8618e341f8f"
+TRELLO_LIST_ID_DONE = "609392e0aaa6e8618e341f91"
```

Comment:
It's nice to store these in the .env file as well, and add instructions for getting the correct IDs from Trello to your readme - that way a new user just has to update .env to get running rather than changing any of the .py files.